calcom/cal.com
# fix: email verification template shows default line heights for texts

## What does this PR do?



- Fixes #16723

## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?



- Make a booking that requires scheduler verification.
- In the verification email sent to the scheduler, inspect the code in the html structure.
- In the applied styles, observe that now no line-height is active. Earlier a fixed line-height of 6px was active on this element, which most likely was the cause of the code being masked or rendered unreadable when viewed on some browsers.
